### PR TITLE
VAGOV-1773 Add assertion for response code and to check for entity da…

### PR DESCRIPTION
**Pull request by schiavo**
…ta count

### CODE REVIEW

File: tests/phpunit/GraphQLTest.php [hunk 1]
```diff
@@ -23,6 +23,8 @@ public function testGraphql($query) {
     $decodedText = html_entity_decode($json_string);
     $entities = json_decode($decodedText, TRUE)['data']['nodeQuery']['entities'];
 
+    $this->assertGreaterThan('0', count($entities), 'No entity data wes returned from request');
+
     foreach ($entities as $entity) {
       if (is_array($entity)) {
         $this->assertArrayHasKey('title', $entity, 'Returned GraphQL does not contain title');
```
Comment (ElijahLynn) [inline tests/phpunit/GraphQLTest.php]: Can you adjust the asserts below this to have the right contain 'text'. eg. fieldIntroText, fieldContentBlock

File: tests/phpunit/GraphQLTest.php [hunk 2]
```diff
@@ -23,6 +23,8 @@ public function testGraphql($query) {
     $decodedText = html_entity_decode($json_string);
     $entities = json_decode($decodedText, TRUE)['data']['nodeQuery']['entities'];
 
+    $this->assertGreaterThan('0', count($entities), 'No entity data wes returned from request');
```
Comment (ElijahLynn) [inline tests/phpunit/GraphQLTest.php]: s/wes/was

Comment (ElijahLynn) [reply tests/phpunit/GraphQLTest.php]: Does the metalsmith query need to get updated here too then? It most likely will be getting empty data too. @ahay-agile6 @acabouet

**Comment by mbenziane** [comment]
Created an environment test-vagovcms#pr-170<br><br> [button_visit-site](http://pr-170.va.agile6.com)

**Comment by ElijahLynn** [comment]
This isn't working for me locally.

**Comment by ElijahLynn** [comment]
This query @ahay-agile6 helped me with works:
```
{
  nodeQuery(limit: 1,filter: {conditions: [{field: "type", value: "page"}]}) {
    count
    entities {
      ... on NodePage {
        nid
        entityBundle
        entityPublished
        title
        fieldIntroText
        fieldContentBlock {
          targetId
          targetRevisionId
        }
      }
    }
  }
}

```

**Comment by ElijahLynn** [comment]
@andyhawks I pushed 2 commits because I think @schiavo is offline. Thumbs up :+1: from me to merge.

**Comment by ElijahLynn** [comment]
build

**Comment by mbenziane** [comment]
Created an environment test-vagovcms#pr-170<br><br> [button_visit-site](http://pr-170.va.agile6.com)

**Comment by ElijahLynn** [comment]
build

**Comment by ElijahLynn** [comment]
build

**Comment by mbenziane** [comment]
rebuild

**Comment by ElijahLynn** [comment]
build

**Comment by mbenziane** [comment]
Created an environment test-vagovcms#pr-170<br><br> [button_visit-site](http://pr-170.va.agile6.com)

**Comment by ElijahLynn** [comment]
build

**Comment by mbenziane** [comment]
rebuild

**Comment by ElijahLynn** [comment]
build

**Comment by ElijahLynn** [comment]
build

**Comment by ElijahLynn** [comment]
build

**Comment by ElijahLynn** [comment]
rebuild

**Comment by mbenziane** [comment]
rebuild

**Comment by mbenziane** [comment]
rebuild

**Pull request merged**